src/renderer/pages/UploaderConfigPage.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div id="config-list-view">
+    <el-row :gutter="20" align="middle" class="config-list">
+      <div class="view-title">
+        {{ $T('SETTINGS') }}
+      </div>
+      <el-col
+        class="config-item-col"
+        v-for="item in curConfigList"
+        :key="item._id"
+        :span="11"
+        :offset="1"
+      >
+        <div
+          :class="`config-item ${defaultConfigId === item._id ? 'selected' : ''}`"
+          @click="() => selectItem(item._id)"
+        >
+          <div class="config-name">{{item.name}}</div>
```

Comment:
Should provide a way to change the config name

Reply:
Somethine like this:

![image](https://user-images.githubusercontent.com/12621342/202598618-d338ecb1-8f28-43e9-83f2-b43c2e50a511.png)